android/src/main/java/com/horcrux/svg/RenderableView.java

```diff
@@ -254,15 +254,9 @@ public void setStrokeOpacity(float strokeOpacity) {
   }
 
   public void setStrokeDasharray(Dynamic dynamicStrokeDasharray) {
-    if (!dynamicStrokeDasharray.isNull()) {
-      ReadableArray strokeDasharray = dynamicStrokeDasharray.asArray();
-      int fromSize = strokeDasharray.size();
-      this.strokeDasharray = new SVGLength[fromSize];
-      for (int i = 0; i < fromSize; i++) {
-        this.strokeDasharray[i] = SVGLength.from(strokeDasharray.getDynamic(i));
-      }
-    } else {
-      this.strokeDasharray = null;
```

Comment:
We want the `else` clause to stay there for sure so it is possible to remove the `strokeDasharray`.